src/System.Buffers.Primitives/System/Buffers/ReadOnlyBuffer.cs

```diff
@@ -98,18 +98,6 @@ public BufferHandle Pin()
             }
         }
 
-        public bool TryGetArray(out ArraySegment<T> buffer)
```

Comment:
I removed it because it is a safe API that gives r/w access to read-only buffer.